### PR TITLE
Mailing list sign up form

### PR DESCRIPTION
**Pull request by garrying**
Follow up to ournetworks/2018#19
- [x] Fix styling for inputs
- [x] Add confirmation state on successful form submit and redirect from formspree

### CODE REVIEW

**Review by dcwalk — COMMENTED** [review]
Layout looks nice!

Just the styling bits (and maybe that button height looks a little overpowering!

**Comment by garrying** [comment]
@dcwalk Made a few updates since last review: ported over form input styles, and the confirmation success message can be previewed at `localhost:4000/?signup=confirmed`. Aiming to merge today

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+document.addEventListener('DOMContentLoaded', function(){
+  if (window.location.search == '?signup=confirmed') {
+    var post = document.querySelector('.header');
+
+    var confirmationEle = document.createElement('div');
+    confirmationEle.className = 'announcement';
+    confirmationEle.setAttribute('role', 'alert');
+    confirmationEle.innerHTML = '<div class="container"><p>Successfully signed up for updates</p></div>';
```
Comment (dcwalk) [inline js/main.js:8]: I'm not getting this @garrying ! tested in a few browsers. does this work for you? 

I feel like my brain is broken atm, but I think i switched something with the page layouts? or is this even loading?

Comment (dcwalk) [reply js/main.js:8]: OMG! I'm silly. I was actually getting redirected from formspree to http://ournetworks.ca/?signup=confirmed so of course it wasn't showing local bits. Okay. nvm.

**Review by dcwalk — APPROVED** [review]
looks good but I'm dying to break out from that blue :)) 💅 
'cmon new look

**Pull request merged**